pkg/server/routes.go

```diff
@@ -215,6 +216,9 @@ func (s *Server) routes(
 	}}
 	gqlConfig := generated.Config{Resolvers: resolver, Directives: gqlDirectives}
 	graphqlServer := handler.NewDefaultServer(generated.NewExecutableSchema(gqlConfig))
+	graphqlServer.Use(extension.FixedComplexityLimit(1000))
```

Comment:
This is required for the complexity to be calculated. 1000 should be large enough to not prevent any of our current queries.